config/notification_banner.yml

```diff
@@ -0,0 +1,12 @@
+# example notification
+# localhost:
+#   notification:
+#     message: Insery message here
+#     date_from: "2024-08-27T18:00:00"
+#     date_to: "2024-08-28T19:00:00"
+
+production:
+  notification:
+    message: This service will be down on Wednesday 27th August between 6 and 8pm for security updates
+    date_from: "2024-08-27T18:00:00"
+    date_to: "2024-08-28T19:00:00"
```

Comment:
should be 2024-08-28T19:00:00 - we agreed 2 hours i think. in case we can do app store